x-pack/legacy/plugins/siem/server/graphql/network/schema.gql.ts

```diff
@@ -33,6 +33,41 @@ export const networkSchema = gql`
     number: Float
   }
 
+  type TopCountriesItemSource {
+    country: String
+    destination_ips: Float
+    flows: Float
+    location: GeoItem
+    source_ips: Float
+  }
+
+  type TopCountriesItemDestination {
+    country: String
+    destination_ips: Float
+    flows: Float
+    location: GeoItem
+    source_ips: Float
+  }
+
+  type NetworkTopCountriesItem {
+    _id: String
+    source: TopCountriesItemSource
+    destination: TopCountriesItemDestination
+    network: TopNFlowNetworkEcsField
```

Comment:
As above, if re-using these types we may want to rename them to be more generic. These types propagate pretty far throughout the codebase, and it becomes difficult to know if it's intended or a mis-type, especially once you get into Redux land.